### PR TITLE
Improve error handling (free use case)

### PR DESCRIPTION
**Pull request by aenario**

### CODE REVIEW

File: client/app/stores/account_store.coffee [hunk 1]
```diff
@@ -141,6 +141,7 @@ class AccountStore extends Store
 
         handle ActionTypes.NEW_ACCOUNT_ERROR, (error) ->
             _newAccountWaiting = false
+            error.uniq = Math.random()
```
Comment (jsilvestre) [inline client/app/stores/account_store.coffee]: Where do you use .uniq afterwards?

Comment (aenario) [inline client/app/stores/account_store.coffee]: nowhere, it just force Panel.shouldComponentUpdate to re-render

Comment (jsilvestre) [inline client/app/stores/account_store.coffee]: Okay, I see. Could you please add a comment about it? I feel like it's not trivial.

**Comment by aenario** [comment]
done

**Comment by aenario** [comment]
added some travis fixes.

**Pull request merged**